releases/latest/beta/Dockerfile.ubuntu.openjdk11

```diff
@@ -1,7 +1,7 @@
 FROM ibm-semeru-runtimes:open-11-jre-focal
 ARG LIBERTY_VERSION=22.0.0.2-beta
```

Comment:
beta version should be updated. I recall that build would catch if the SHA value doesn't match. Is that still the case?